packages/material-ui/src/AlertTitle/AlertTitle.js

```diff
@@ -43,7 +43,7 @@ const AlertTitle = React.forwardRef(function AlertTitle(inProps, ref) {
 
   const { className, ...other } = props;
 
-  const styleProps = other;
+  const styleProps = { ...props };
```

Comment:
Why do we spread it? I thought that `styleProps` was read-only.

Reply:
I was trying to fix this https://app.circleci.com/pipelines/github/mui-org/material-ui/36617/workflows/d3d9d9e5-7eb1-4bd3-a9ae-ce6858c3edd0/jobs/217645

Reply:
Please add a comment explaining why. This should probably be a TODO for `buildAPI`